app/Repositories/UserRepository.php

```diff
@@ -112,6 +113,23 @@ public function counts()
     }
 
     /**
+     * Get blog authors, their number of blogs, the last created date, and title of their latest blog entry.
+     *
+     * @return Illuminate\Database\Eloquent\Builder
+     */
+    public function getAuthorPostsWithOrder()
+    {
+        $authors = $this->model
+            ->join('posts', 'posts.user_id', '=', 'users.id')
+            ->select(DB::raw('count(posts.id) as total_count'), 'username', 'posts.id', 'posts.created_at', 'posts.title')
```

Comment:
Ah, I added this tho: https://github.com/aphofstede/laravel5-3-example/pull/1/files#diff-7148cca22aa02ecc66532f4c02d2f3f7R7